etc/systemd/system/zfs-volume.service.in

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Wait for ZFS Volume (zvol) links in /dev
+DefaultDependencies=no
+After=systemd-udev-settle.service
+After=zfs-import.target
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/usr/lib/zfs-linux/zvol-wait
```

Comment:
You might not need to specify a full path here if it's installed in `/usr/lib/systemd/scripts/`, otherwise you'll need to substitute `@systemdscriptsdir@` and update the `*.in` rule in the `Makefile.am` to do the substitution.